README.md

```diff
@@ -8,25 +8,48 @@ We are dealing with a high volume of requests, but we are doing our best to stay
 
 ## Development
 
-To build the application:
+1. Clone the repo:
 
-1. Install [Node JS](https://nodejs.org/en/download/)
-2. Install [Yarn](https://classic.yarnpkg.com/en/docs/install)
-3. Install [Gatsby](https://www.gatsbyjs.org/docs/gatsby-cli/)
```

Comment:
- we want to use the same version of node as in .nvmrc
- we want to use the version of gatsby in yarn.lock

future: we can pin yarn version too